src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/concat_const_inplace.cpp

```diff
@@ -70,18 +75,23 @@ class ConcatConstantInPlaceTest : public testing::WithParamInterface<ov::element
 
 namespace {
 TEST_P(ConcatConstantInPlaceTest, smoke_ConcatConstantInPlaceTest_CPU) {
+    if (this->GetParam() == ov::element::f16) {
+        if (!(ov::with_cpu_x86_avx512_core_fp16() || ov::with_cpu_x86_avx512_core_amx_fp16())) {
+            GTEST_SKIP() << "Skipping test, platform don't support precision f16";
+        }
+    }
```

Comment:
Concat node doesn't have any dependency on ISA. What is the purpose of this check?

Reply:
already removed them